identity/src/main/java/com/stripe/android/identity/IdentityOnBackPressedHandler.kt

```diff
@@ -84,6 +88,21 @@ internal class IdentityOnBackPressedHandler(
         }
     }
 
+    private fun showAboutToCancelAlert() {
+        val builder = AlertDialog.Builder(context)
+        builder.setMessage(R.string.stripe_identity_confirm_cancel)
+            .setPositiveButton(R.string.stripe_identity_yes) { _, _ ->
+                finishWithCancelResult(
+                    identityViewModel,
+                    verificationFlowFinishable,
+                    destination?.route?.routeToScreenName()
+                        ?: IdentityAnalyticsRequestFactory.SCREEN_NAME_UNKNOWN
+                )
+            }.setNegativeButton(R.string.stripe_identity_no) { _, _ ->
+            }
+        builder.show()
+    }
```

Comment:
I don't love that this is called `showAboutToCancelAlert`, but actually has a very specific action that happens on confirm.

What do you think about making this a more generic method? Instead of hard-coding in the actions on "yes" and "no", it could be a library method that takes in functions for both cases.

This would promote better re-use, and would make the method name more applicable to what it actually does.

Reply:
Created a util for alerts: https://github.com/stripe/stripe-android/pull/7051/commits/b43c1b8f58e300e757bb5a53e53ba2078afc3003